thesslstore_module.php

```diff
@@ -1480,6 +1480,43 @@ public function addService($package, array $vars=null, $parent_package=null, $pa
     }
 
     /**
+     * Edits the service on the remote server. Sets Input errors on failure,
+     * preventing the service from being edited.
+     *
+     * @param stdClass $package A stdClass object representing the current package
+     * @param stdClass $service A stdClass object representing the current service
+     * @param array $vars An array of user supplied info to satisfy the request
+     * @param stdClass $parent_package A stdClass object representing the parent
+     *  service's selected package (if the current service is an addon service)
+     * @param stdClass $parent_service A stdClass object representing the parent
+     *  service of the service being edited (if the current service is an addon service)
+     * @return array A numerically indexed array of meta fields to be stored for this service containing:
+     *  - key The key for this meta field
+     *  - value The value for this key
+     *  - encrypted Whether or not this field should be encrypted (default 0, not encrypted)
+     * @see Module::getModule()
+     * @see Module::getModuleRow()
+     */
+    public function editService($package, $service, array $vars = null, $parent_package = null, $parent_service = null)
+    {
+        // Set fields to update locally
+        $service_fields = ['thesslstore_token', 'thesslstore_order_id', 'thesslstore_fqdn'];
+        foreach ($service_fields as $field) {			
+			// check if the field is set and is not empty
```

Comment:
Formatting is off here with white space usage. You should use spaces rather than tabs (1 tab = 4 spaces).